lisa/features/gpu.py

```diff
@@ -122,8 +122,7 @@ def install_compute_sdk(self, version: str = "") -> None:
         self._install_gpu_dep()
         try:
             # install LIS driver if required and not already installed.
-            if LisDriver.can_install:
-                self._node.tools[LisDriver]
+            self._node.tools[LisDriver]
         except Exception as identifier:
             self._log.debug(
                 "LisDriver is not installed. It might not be required. " f"{identifier}"
```

Comment:
remove the `" f"`. It doesn't need in one line.

Reply:
removed